rubix-tests/src/test/java/com/qubole/rubix/tests/TestNonLocalReadRequestChain.java

```diff
@@ -323,13 +324,16 @@ private void stopAllServers()
     if (bookKeeperServer != null) {
       bookKeeperServer.stopServer();
     }
-    LocalDataTransferServer.stopServer();
+    if (LocalDataTransferServer.isServerUp()) {
```

Comment:
can't we add this condition LocalDataTransferServer.isServerUp() in isServerUp() method itself?

Reply:
Yes, changing